### PR TITLE
refactored doc quality transform as its own module with its own dpk_ namespace

### PR DESCRIPTION
**Pull request by touma-I**
## Why are these changes needed?

This is a continuation of the ongoing effort to restructure each transform built as its own module (e.g. dpk_doc_quality) with a ray submodule (dpk_dock_quality.ray ).

Removed python and ray folders and redo  Dockerfile.python and Dockerfile.ray using new structure
remove pyproject.toml and Makefiles
move python code under dpk_doc_quality
move ray code under dpk_doc_quality/ray
change import statement to include module name
replace recursive Makefile and use targets from .make.cicd.targets
adapt kfp_ray/Makefile and other make target

## Related issue number (if any).

#774

### CODE REVIEW

File: transforms/language/doc_quality/README.md [hunk 1]
```diff
@@ -1,11 +1,175 @@
...
+| docq_alphabet_word_ratio | float | the ratio of words having at least one alphabetic character | ALL |
+| docq_contain_common_en_words | bool | whether the given `text` contains common English words like `the`, `and`, `to`, `that`, `of`, `with`, `be`, and `have`| ALL |
+| docq_avg_ja_sentence_len | int | average sentence length for an input text, inspired by an OSS HojiChar. | ja |
+| docq_first_ja_alphabet_pos | int | first position of occurrence of Japanese alphabets (i.e., Hiragana or Katakana) | ja |
+
+You can see more detailed backgrounds of some columns in [Deepmind's Gopher paper](https://arxiv.org/pdf/2112.11446.pdf)
+
+## Configuration
+
+The set of dictionary keys holding [DocQualityTransform](src/doc_quality_transform.py)
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Need to fix link

File: transforms/language/doc_quality/README.md [hunk 2]
```diff
@@ -1,11 +1,175 @@
...
+
+### Transforming data using the transform image
+
+To use the transform image to transform your data, please refer to the 
+[running images quickstart](../../../../doc/quick-start/run-transform-image.md),
+substituting the name of this transform image and runtime as appropriate.
+
+## Testing
+
+Following [the testing strategy of data-processing-lib](../../../../data-processing-lib/doc/transform-testing.md)
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Correct link?

File: transforms/language/doc_quality/README.md [hunk 3]
```diff
@@ -1,11 +1,175 @@
...
+
+Document Quality configuration and command line options are the same as for the base python transform. 
+
+### Running
+
+#### Launched Command Line Options 
+When running the transform with the Ray launcher (i.e. TransformLauncher),
+In addition to those available to the transform as defined in [here](../python/README.md),
+the set of 
+[ray launcher](../../../../data-processing-lib/doc/ray-launcher-options.md) are available.
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Correct link?

File: transforms/language/doc_quality/README.md [hunk 4]
```diff
@@ -1,11 +1,175 @@
...
+
+
+### Configuration and command line Options
+
+Document Quality configuration and command line options are the same as for the base python transform. 
+
+### Running
+
+#### Launched Command Line Options 
+When running the transform with the Ray launcher (i.e. TransformLauncher),
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Need to fix documentation

File: transforms/language/doc_quality/README.md [hunk 5]
```diff
@@ -1,11 +1,175 @@
...
+In addition to those available to the transform as defined in [here](../python/README.md),
+the set of 
+[ray launcher](../../../../data-processing-lib/doc/ray-launcher-options.md) are available.
+
+#### Running the samples
+To run the samples, use the following `make` targets
+
+* `run-cli-sample` - runs src/doc_quality_transform.py using command line args
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Need to fix path

File: transforms/language/doc_quality/README.md [hunk 6]
```diff
@@ -1,11 +1,175 @@
...
+```
+Then 
+```shell
+ls output
+```
+To see results of the transform.
+
+### Code example
+
+[notebook](../doc_quality.ipynb)
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Need to fix link

File: transforms/language/doc_quality/README.md [hunk 7]
```diff
@@ -1,11 +1,175 @@
...
+
+### Transforming data using the transform image
+
+To use the transform image to transform your data, please refer to the 
+[running images quickstart](../../../../doc/quick-start/run-transform-image.md),
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Correct link?

File: transforms/language/doc_quality/README.md [hunk 8]
```diff
@@ -1,11 +1,175 @@
...
+```shell
+ls output
+```
+To see results of the transform.
+
+
+#### Transforming data using the transform image
+
+To use the transform image to transform your data, please refer to the 
+[running images quickstart](../../../../doc/quick-start/run-transform-image.md),
```
Comment (dtsuzuku-ibm) [inline transforms/language/doc_quality/README.md]: Correct link?

**Review by dtsuzuku-ibm — COMMENTED** [review]
Thanks.

Needs some update on README.md

**Comment by shahrokhDaijavad** [comment]
@touma-I I fixed the README file.

**Comment by touma-I** [comment]
> @touma-I I fixed the README file.

@shahrokhDaijavad Million thanks. @dtsuzuku-ibm Can you please do one more pass and approve the PR. I need you to approve it before I can merge it. Thanks

**Review by dtsuzuku-ibm — APPROVED** [review]
LGTM

**Pull request merged**